### ISSUE TITLE
Error when running on simulator

### ISSUE THREAD

**Issue by obromios** [ryanmcdermott/react-native-login#3]
I have installed your app as instructed. When I press run, it builds successfully, but when I reset the simulator (cmd/r), the simulator hangs with a blank screen and there is the following error in the console.

```
[tid:com.facebook.React.RCTSourceCodeQueue][RCTModuleMethod.m:381] RCTSourceCode.getScriptText was called with 0 arguments, but expects 2
(lldb) 
```

Your app looks very useful, and seems to be close to working...

**Comment by obromios** [comment]
It turns out that I had not followed your instructions properly, I was not installing npm twice, once in the project and once in the server. When I did this, this problem disappeared, though the next problem is finding react-native-cookies. Anyway I will close this problem and will continue work on the react-native-cookies problem.

**Comment by obromios** [comment]
I was not able to get your app running, even when I followed your instructions closely. I suspect that it was due to an out of date react-native original installation.  In the end, what I did was just copy and pasted your App/components code into my own fresh react-native app, and all worked well.

I noticed two deprecation warnings to do with your WebView code. Evidently, `url` has been replaced by a `source` method and `javaScriptEnabledAndroid` has been replaced by `javaScriptEnabled`.  I modified the code as shown below, and the deprecation warning disappeared. I will re-open this issue, just to make sure you see this post.  This repository has been very useful, thank you.

```
      <View style={[styles.container]}>
        <WebView
          ref={'webview'}
          automaticallyAdjustContentInsets={false}
          style={styles.webView}
          source={{uri: LOGIN_URL}}
          javaScriptEnabled={true}
          onNavigationStateChange={this.onNavigationStateChange.bind(this)}
          startInLoadingState={true}
          scalesPageToFit={true}
        />
      </View>
```
